### PR TITLE
Fix for tinymce tests

### PR DESCRIPTION
**Pull request by lyralemos**
This PR fix tinymce tests falling since #849.

### CODE REVIEW

**Comment by lyralemos** [comment]
If the changes you mentioned in #854 fixes tinymce tests I see no reason why not to discard this PR.

What you said about i18n.currentLanguage being TinyMCE specific, actually the changes introduced in #849 fixes both tinyMCE and Folder Contents. This happens because all country specific languages codes in plone.app.locales are like pt_BR, for instance, and not like pt-br (which is the standard for html languages codes). For that reason I see no problem with transforming the language code.

**Comment by petschki** [comment]
@lyralemos thanks for clarification. Then it's ok for me to transform the language code here. I wasn't aware of plone.app.locales country-language code pattern. So we actually can merge #854 and discard this PR.

**Comment by petschki** [comment]
#854 with test fix is merged .... closing this

File: mockup/js/i18n.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ define([
     self.currentLanguage = $('html').attr('lang') || 'en-us';
```
[the remark this answers is not in the document]
Comment (davilima6) [reply mockup/js/i18n.js]: Sounds good to me

**Comment by davilima6** [comment]
@lyralemos, I think the web standard is IETF ('pt-BR'): https://www.w3.org/International/questions/qa-html-language-declarations

That's also the format you get when you visit one of Google or Mozilla websites so I suggest we should stick to it. And change our `<html lang>` formatting.

There's also POSIX ('pt_BR') to use for filenames, which TinyMCE does (unfortunately in a folder misnamed `langs`, since a locale is the sum of language and region). OTOH in Moment.js, folder is correctly named "locales" but files are IETF and lowercased. Maybe I miss something, if not looks like an inconsistency to me.

In any case it'd be useful to have methods in `mockup-i18n` delivering each of these versions (not 100% sure about the lower case version, if it's an anti-pattern we shouldn't promote it). What do you think?

/cc @jensens @agitator @ale-rt @petschki

**Pull request closed without merging**